### PR TITLE
Merge release/19.8 — 19.8-rc-2 + 19.7.1 hotfix + Release Notes

### PR DESCRIPTION
**Pull request by AliSoftware**
Merging the release branch after doing a new beta.

### The new beta includes

 * Fixes from `release/19.7.1` hotfix back-merged into `release/19.8` (https://github.com/wordpress-mobile/WordPress-Android/pull/16475):
    * https://github.com/wordpress-mobile/WordPress-Android/pull/16459
    * https://github.com/wordpress-mobile/WordPress-Android/pull/16470
 * ⚠️  Latest `strings.xml` from GlotPress — not many `strings.xml` were downloaded in practice due to an ongoing issue with GlotPress rate-limiting
    * See https://github.com/wordpress-mobile/release-toolkit/pull/361
    * See [thread in WordPress.org Slack](https://wordpress.slack.com/archives/C02RP50LK/p1651640726217119?thread_ts=1651607938.132789&cid=C02RP50LK)
 * Version bump in `version.properties` —
    * Manually adjusted in c8d07241ac1917e2097df5fc53b6af119aa472ef to account for `1211` versionCode already being consumed by `19.7.1` hotfix

### This PR also includes

 * Editorialized Release Notes for 19.8 — so that they start getting imported into GlotPress and be translated.

### CODE REVIEW

File: fastlane/Pluginfile
Comment (AliSoftware) [inline fastlane/Pluginfile:10]: This was in order to get https://github.com/wordpress-mobile/release-toolkit/pull/361 in, in hope that it would reduce if not completely fix the `429 Too Many Requests` HTTP errors when downloading translations from GlotPress.

Alas, that didn't fix them, which is why there are only a handful of `strings.xml` files updated in that PR — as the others hit that 429 HTTP error. We're still working with GP.org folks to solve that ideally before next beta (and absolutely before next Wednesday's beta for localized testing and Friday's final build); but in the meantime that just means reduced translations in that intermediate beta, which is not that big a deal… for now.

File: fastlane/lanes/release.rb
Comment (AliSoftware) [inline fastlane/lanes/release.rb:92]: I added that quickly so that it would pause and give me the opportunity to manually fix the `versionCode` bumps (in c8d07241ac1917e2097df5fc53b6af119aa472ef) to account for the `19.7.1` hotfix consuming the code `1211`.

I might actually keep it for the future, even if it's mostly useful for hotfixes, it doesn't hurt to have it.

**Review by guarani — COMMENTED** [review]
I confirmed that this includes the code changes from https://github.com/wordpress-mobile/WordPress-Android/pull/16470 and I downloaded the builds to confirm that Plans is no longer visible. I don't feel qualified to provide a full PR review though, so I'll have to leave that to another reviewer 🙇.

**Review by ashiagr — APPROVED** [review]
Changes from #16459 are included in this PR. Thanks!

File: fastlane/Pluginfile
Comment (mokagio) [reply fastlane/Pluginfile:10]: I fixed a small typo in this comment, which previously had `439` instead of `429`. 

I'm posting a note about it because I went on a 439 status code tangent... Found something cool: [HTTP has no official 439 status code](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#client_error_responses), however, the [folks at Etsy use 439 for "non-artisanal" requests](https://twitter.com/lozzd/status/380454656243412992?s=20&t=hEkONukHPICFDj24je3j_A) 🤔 😄

**Pull request merged**
